jbake-core/src/main/java/org/jbake/app/ContentStore.java

```diff
@@ -221,22 +236,25 @@ public void insertSignature(String currentTemplatesSignature) {
     }
 
     private DocumentList query(String sql) {
+        activateOnCurrentThread();
         List<ODocument> results = db.query(new OSQLSynchQuery<ODocument>(sql));
         return DocumentList.wrap(results.iterator());
     }
 
     private DocumentList query(String sql, Object... args) {
+        activateOnCurrentThread();
         List<ODocument> results = db.command(new OSQLSynchQuery<ODocument>(sql)).execute(args);
         return DocumentList.wrap(results.iterator());
     }
 
     private void executeCommand(String query, Object... args) {
+        activateOnCurrentThread();
         db.command(new OCommandSQL(query)).execute(args);
     }
 
     public Set<String> getTags() {
         DocumentList docs = this.getAllTagsFromPublishedPosts();
-        Set<String> result = new HashSet<String>();
+        Set<String> result = new HashSet<>();
```

Comment:
Is this an intentional change? I guess not.

Reply:
This is intentional. Java 7 introduced "Type Inference" and the Diamond operator.
https://docs.oracle.com/javase/tutorial/java/generics/genTypeInference.html

It's also a code smell at sonarqube:

```
Java 7 introduced the diamond operator (<>) to reduce the verbosity of generics code. For instance, instead of having to declare a List's type in both its declaration and its constructor, you can now simplify the constructor declaration with <>, and the compiler will infer the type.
```